tests/integration/test_catalogs.py

```diff
@@ -37,6 +37,104 @@ async def test_catalogs(client: _TestClient, mocker: MockFixture) -> None:
+    assert 0 < len(g), "*/* has no triples"
+
+    headers = MultiDict([(hdrs.ACCEPT, "*/*")])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status, "*/* failed"
+    assert "text/turtle; charset=utf-8" == response.headers["Content-Type"]
+    data = await response.text()
+    g = Graph().parse(data=data, format="text/turtle")
+    assert 0 < len(g), "*/* has no triples"
+
+    headers = MultiDict(
+        [(hdrs.ACCEPT, "text/turtle"), (hdrs.ACCEPT, "application/ld+json")]
+    )
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status, "Multiple headers failed"
+    assert "text/turtle; charset=utf-8" == response.headers["Content-Type"]
+    data = await response.text()
+    g = Graph().parse(data=data, format="text/turtle")
+    assert 0 < len(g), "*/* has no triples"
+
+    headers = MultiDict([(hdrs.ACCEPT, "not/acceptable")])
+    response = await client.get("/catalogs", headers=headers)
+    assert 406 == response.status, "not/acceptable failed"
+
+
+@pytest.mark.integration
+async def test_catalogs_content_negotiation(
+    client: _TestClient, mocker: MockFixture
+) -> None:
+    """Should return 200 and correct content-type."""
+    # Set up the mock
+    mocker.patch(
+        "SPARQLWrapper.SPARQLWrapper.queryAndConvert",
+        return_value=_mock_query_and_convert_result(),
+    )
+
+    headers = MultiDict([(hdrs.ACCEPT, "text/turtle, application/ld+json")])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status
+    assert (
+        "text/turtle; charset=utf-8" == response.headers["Content-Type"]
+    ), " Content-Type in response-header should be text/turtle."
+
+    headers = MultiDict([(hdrs.ACCEPT, "application/ld+json, text/turtle")])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status
+    assert (
+        "application/ld+json; charset=utf-8" == response.headers["Content-Type"]
+    ), " Content-Type in response-header should be application/ld+json."
+
+    headers = MultiDict([(hdrs.ACCEPT, "not/acceptable, */*")])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status, "'not/acceptable,*/*' failed"
+
+    headers = MultiDict([(hdrs.ACCEPT, "text/plain,*/*")])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status, "'text/plain,*/*' failed"
+    assert (
+        "text/turtle; charset=utf-8" == response.headers["Content-Type"]
+    ), " Content-Type in response-header should be text/turtle."
+
+    headers = MultiDict([(hdrs.ACCEPT, "*/*,text/plain")])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status, "'*/*,text/plain' failed"
+    assert (
+        "text/turtle; charset=utf-8" == response.headers["Content-Type"]
+    ), " Content-Type in response-header should be text/turtle."
+
```

Comment:
Kan du teste denne? `application/ld+json` er mer spesifik, og skal derfor velges. Se https://www.rfc-editor.org/rfc/rfc9110.html#section-12.5.1
```suggestion

    headers = MultiDict([(hdrs.ACCEPT, "*/*, application/ld+json")])
    response = await client.get("/catalogs", headers=headers)
    assert 200 == response.status, "'*/*, application/ld+json' failed"
    assert (
        "application/ld+json; charset=utf-8" == response.headers["Content-Type"]
    ), " Content-Type in response-header should be application/ld+json."
```